C++/test/algorithm/string/heaps_algorithm.cpp

```diff
@@ -0,0 +1,41 @@
+#define CATCH_CONFIG_MAIN
+#include "third_party/catch.hpp"
+#include "algorithm/string/heaps_algorithm.hpp"
+
+
+TEST_CASE("Base cases", "[string][heaps_algorithm]") {
+    vector<string> output_1;
+    string s = ""; // empty string
```

Comment:
Again, personal preference, but I'd prefer `str` over `s`

Reply:
I prefer `s` over `str` because `str()` is a standard library function. See https://www.cplusplus.com/reference/sstream/stringstream/str/

Reply:
Of course. Thanks. 🙂